### PR TITLE
Raising the Bar chapter updates

### PR DESCRIPTION
**Pull request by SamQAie**
I've updated Raising the Bar chapter. 
Some new paragraphs has been added and minor changes has been applied to some of the existing ones. Also structure has been updated.

### CODE REVIEW

File: RaisingTheBar.asciidoc [hunk 1]
```diff
@@ -1,34 +1,70 @@
...
+
+For this reason, the Test Pilot is an integral member of the team who will be writing test cases when the first line of code is being written.
 
 === The importance of acceptance criteria
 
-In order to ensure that each and every Story can be validated, a User Story should have an acceptance criteria to make it super clear to the developer what the business are accepting, and how the Test Pilot will be testing your solution. Let’s get the “story” straight before we start with delivering value as to what everybody’s expectations are. An example acceptance criteria for bug resolution could be:
+In any successful software project, a well-planned testing strategy is key. We define testing not only by deadlines and requirements but by *acceptance criteria* agreed upon with stakeholders. It’s critical to create clear, *SMART acceptance criteria*—Specific, Measurable, Achievable, Relevant, and Time-bound—to ensure shared expectations across teams. This allows us to have precise testing and reduces ambiguity in what is deemed 'done.'
```
Comment (BenGWeeks) [inline RaisingTheBar.asciidoc:15]: Can we create a section for this "Test Strategy" section? I think this could also be extended upon.

Comment (BenGWeeks) [reply RaisingTheBar.asciidoc:15]: What does *SMART acceptance criteria*—Specific mean?

File: RaisingTheBar.asciidoc [hunk 2]
```diff
@@ -1,34 +1,70 @@
...
+=== Continuous Feedback and Improvement
+
+Effective *defect management* and continuous feedback loops between QA and development teams ensure that issues are addressed promptly. So keep in mind that a streamlined process for tracking defects, setting priorities, and reporting results is vital. Regular reviews of key metrics such as defect density and mean time to resolution (MTTR) are really nice tools for assessing the health of the project and making informed decisions for continuous improvement.
+
+=== AI in QA
+
+As QA continues to evolve, *AI and machine learning* are becoming key players in advancing the field. From automating test case generation to predictive analytics for identifying high-risk areas in code, AI can augment traditional QA practices. Exploring these technologies will help future-proof testing processes and drive even more robust software quality.
 
 === Static code analysis
```
Comment (BenGWeeks) [inline RaisingTheBar.asciidoc:41]: I think we need a main "Types of Testing" section, and then under that we can run through the different types of testing. We could probably re-use the text in the test strategy template that explains the different types of testing. For example, we need "Chaos Monkey" testing in there :-)

File: RaisingTheBar.asciidoc [hunk 3]
```diff
@@ -1,34 +1,70 @@
...
+
+A/B testing also allows high performance sites to always be available with zero warm up time needed. You deploy to the staging slot, start the slot running, and once running, swap this slot to be the production slot.
+
+You can also look at implementing an intelligent solution that will analyse the logs and dynamically amend the routing rules to pass users to the new code on confirmation that there are no detrimental issues affecting performance or correct behaviour. Or, alternatively, fall back to the old code if an issue is detected.
+
+=== Automated UI and API testing
+
+To speed up the repeatable nature of testing to help improve the cadence of release, where possible we should leverage the use of automated test tools such as Playwright or Selenium Web Driver.
+
+=== CI/CD Pipelines
```
Comment (BenGWeeks) [inline RaisingTheBar.asciidoc:61]: Amend the title of this?

File: RaisingTheBar.asciidoc [hunk 4]
```diff
@@ -1,34 +1,70 @@
...
+
+Automation is crucial to maintaining a high level of efficiency and quality in fast-moving projects. Incorporating automated tests into CI/CD pipelines enables early bug detection and helps validate every code change. These tests should span from unit tests to integration and regression tests, ensuring that new updates don't break existing functionality.
+
+=== Exploratory and Context-Driven Testing
 
-To speed up the repeatable nature of testing to help improve the cadence of release, where possible we should leverage the use of automated test tools such as Selenium Web Driver .
+Beyond automation, exploratory testing is invaluable for identifying edge cases and unexpected behavior. Testers use their knowledge of the system to explore and test functionality in unscripted ways, often revealing bugs that standard scripted tests may miss. Coupled with context-driven testing, this approach allows testers to adapt based on the specific requirements and nuances of each project, making the testing process more flexible and responsive.
 
 === Security assessment & compliance
```
Comment (BenGWeeks) [inline RaisingTheBar.asciidoc:70]: Can you also amend the text "Consider security during the security and compliance using automated tools such as ..." (as "security during the security" sounds odd (I know that wasn't you, but while we are addressing this section.

File: RaisingTheBar.asciidoc [hunk 5]
```diff
@@ -50,16 +86,8 @@ Typically you would carry out performance testing in the Test (QA) environment.
 
 === User acceptance testing
```
Comment (BenGWeeks) [inline RaisingTheBar.asciidoc:88]: Can we add to this section about how we have a number of UAT sessions per Sprint. We don't expect to be able to test all the Features in week 1 of the Sprint, but get the Features through week by week. By having 3 sessions in a Sprint, we have 3 opportunities to get to production rather than only 1.

**Review by BenGWeeks — COMMENTED** [review]
Can we take a look through the Test Strategy and cover other items in there. For example, the ability for them to flag a minor bug and it has high priority (forget what we call that).

File: RaisingTheBar.asciidoc (hunk 1 above)
Comment (SamQAie) [reply RaisingTheBar.asciidoc:15]: 1. Done
2. **SMART** means Specific, Measurable, Achievable, Relevant, and Time-bound acceptance criteria. When focusing on Specific in SMART acceptance criteria, it means that the requirement or objective should be clear, detailed, and unambiguous.

File: RaisingTheBar.asciidoc (hunk 2 above)
Comment (SamQAie) [reply RaisingTheBar.asciidoc:43]: Nice idea, done.

File: RaisingTheBar.asciidoc (hunk 4 above)
Comment (SamQAie) [reply RaisingTheBar.asciidoc:80]: Fixed

File: RaisingTheBar.asciidoc (hunk 5 above)
Comment (SamQAie) [reply RaisingTheBar.asciidoc:98]: Done

**Comment by SamQAie** [comment]
Pull Request's Approval is required

File: RaisingTheBar.asciidoc (hunk 1 above)
Comment (BenGWeeks) [reply RaisingTheBar.asciidoc]: Let's say what that is then.

**Comment by BenGWeeks** [comment]
Has a merge conflict that also needs resolving.

**Comment by SamKyselyov** [comment]
@BenGWeeks 
Merge conflict solved

**Comment by BenGWeeks** [comment]
Review this please @claude
